resources/js/app-layout.js

```diff
@@ -194,11 +196,17 @@ if (isMobileDevice) {
     el: "#navbarMobile",
     components: {
       requestModalMobile,
+      WelcomeModal,
     },
     data() {
       return {
       };
     },
+    mounted() {
+      if (this.$cookies.get("firstMounted") === "true") {
+        $("#welcomeModal").modal("show");
```

Comment:
this is like this:
https://bootstrap-vue.org/docs/components/modal#using-thisbvmodalshow-and-thisbvmodalhide-instance-methods

Reply:
i don't know why, but with refs and b-modal it doesn't work